jupyterhub/auth.py

```diff
@@ -316,6 +341,23 @@ def check_blacklist(self, username):
         """
         return True
 
+    def is_admin(self, handler, data, authentication, *args, **kwargs):
```

Comment:
Let's not add `*args, **kwargs` here. I don't think that's a good plan for methods that will be overridden by subclasses, since there's no indicator whether this has been implemented by sublasses until/unless args are added, which is effectively the same as if they weren't here at all on the base class.

Instead, let's take two args:

```suggestion
    def is_admin(self, handler, authenticated):
```

where the second arg is the authenticated user dict.

Reply:
That's fair. I was actually just thinking of that on my drive in today. I'll get #2268 refactored as well.